website/docs/d/is_subnet_reserved_ips.html.markdown

```diff
@@ -50,5 +50,15 @@ In addition to the argument reference list, you can access the following attribu
   - `resource_type` -  (String) The resource type.
   - `target` - (String) The ID of the target for the reserved IP.
   - `target_crn` - (String) The crn of the target for the reserved IP.
+	- `target_reference` - (List) The target this reserved IP is bound to. If absent, this reserved IP is provider-owned or unbound.
```

Comment:
Update the formatting for `target_reference`

Reply:
resolved

Reply:
Is this `target_reference` is under `target_crn` ? 

Check this and let me know ?

Reply:
no

Reply:
updated the file